src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -159,19 +159,25 @@ stop_metadata_sync_to_node(PG_FUNCTION_ARGS)
 
 
 /*
- * ShouldSyncTableMetadata checks if a distributed table has streaming replication model
- * and hash distribution. In that case the distributed table is considered an MX table,
- * and its metadata is required to exist on the worker nodes.
+ * ShouldSyncTableMetadata checks if the metadata of a distributed table should be
+ * propagated to metadata workers, i.e. the table is an MX table or reference table.
+ * Tables with streaming replication model (which means RF=1) and hash distribution are
+ * considered as MX tables while 2PC replicated tables with none distribution are
```

Comment:
Minor: probably no need to mention ` 2PC replicated tables` on the function comment?